2-classes-db/task.md

```diff
@@ -1,79 +1,107 @@
 # Class Database
+
 Run the following commands:
+
 ```
 createdb homework
 psql -d homework -f setup.sql
 ```
+
 This will create and populate a database called homework.
 
 It is based on real data from the government found at
 https://data.gov.uk/dataset/72eaec8e-0d32-4041-a553-87b852abee64/spend-over-25-000-in-worcestershire-acute-hospitals-nhs-trust
 
 You can work with it by running
+
 ```
 psql homework
 ```
+
 ## Submission
 
 Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands after each question.
 
 ### 1. Show the date, transaction_no, description and amount for those transactions whose amount is between £30,000 and £31,000.
-```sql
 
+```sql
+select data,transaction_no, description , amount from spends where amount >30000 and amount <31000;
 ```
```

Comment:
CORRECT - Whilst the question is ambiguous, it is worth pointing out that your query will not include values of 30,000 or 31,000 (which could be included with >= or <= comparisons).  You could also use the BETWEEN operator (see https://commandprompt.com/education/how-to-use-between-operator-in-postgresql/)